src/main/java/com/odysseusinc/athena/config/WebSecurityConfig.java

```diff
@@ -93,7 +108,16 @@ public void configure(WebSecurity webSecurity) throws Exception {
         @Override
         protected void configure(final HttpSecurity http) throws Exception {
 
-            final SecurityFilter filter = new SecurityFilter(config, "HeaderClient");
+            final SecurityFilter filter = new SecurityFilterWithSkipCondition(config, "HeaderClient") {
```

Comment:
But why not to add /api/v1/concepts to http config with permitAll instead of constructing filter?